synapse_invite_checker/invite_checker.py

```diff
@@ -352,12 +361,12 @@ async def user_may_invite(
         # Step 1, check federation allow list
         inviter_id = UserID.from_string(inviter)
         inviter_domain = inviter_id.domain
-        fedlist = await self.fetch_federation_allow_list()
-        if inviter_domain not in fedlist:
+        domain_set, _ = await self.fetch_federation_allow_list()
+        if inviter_domain not in domain_set:
             self.fetch_federation_allow_list.cache_clear()
-            fedlist = await self.fetch_federation_allow_list()
+            domain_set = await self.fetch_federation_allow_list()
```

Comment:
this looks wrong, if you now use a tuple

Reply:
good catch, I'll fix that up

Reply:
Fixed in e64ae8f